docs/src/main/sphinx/connector/hive.rst

```diff
@@ -956,6 +956,13 @@ The following operations are not supported when ``avro_schema_url`` is set:
 Procedures
 ----------
 
+Use the :doc:`/sql/call` procedure to perform data manipulation or
```

Comment:
This should say

Use the CALL statement to perform 


or

Call a procedure to

Reply:
or even 

CALL a procedure to

Reply:
I like the first one, PR is updated to reflect the edit.